Makefile.v8

```diff
@@ -22,21 +26,35 @@ plv8_config.h: $(AUTOV8_LIB)
 
 $(AUTOV8_DIR):
 	mkdir -p build
-	curl -L 'https://github.com/v8/v8/archive/$(AUTOV8_VERSION).tar.gz' \
-		| tar zxf - -C build
+	cd build; fetch v8; cd v8; git checkout $(AUTOV8_VERSION); gclient sync
```

Comment:
I'd love to eliminate this kind of dependency as much as possible to make it smooth for any packagers.